src/modules/template/router.js

```diff
@@ -0,0 +1,27 @@
+const schema = require('./schema')
+const TemplateService = require('./service')
+
+module.exports = async (server, opts) => {
+  const { mongol } = opts
+  const templateService = new TemplateService(mongol)
+
+  server.post('/template', { schema: schema.createTemplate }, async (req, res) => {
+    try {
+      const template = await templateService.createTemplate(req.body)
+      res.status(200).send(template)
+    } catch (err) {
+      req.log.error(err.message)
```

Comment:
Đây cũng là 1 cái cần lưu ý mà t đã nhắc ông sửa trước đây rồi mà ông không nhớ. Ông đâu chắc lỗi này ở request, nên tốt nhất là
```suggestion
      server.log.error(err.message)
```